lib/Doctrine/ORM/Persisters/Entity/AbstractEntityInheritancePersister.php

```diff
@@ -66,15 +66,16 @@ protected function getSelectColumnSQL($field, ClassMetadata $class, $alias = 'r'
      * @param string $tableAlias
      * @param string $joinColumnName
      * @param string $quotedColumnName
+     * @param bool   $isIdentifierColumn
      * @param string $type
      *
      * @return string
      */
-    protected function getSelectJoinColumnSQL($tableAlias, $joinColumnName, $quotedColumnName, $type)
+    protected function getSelectJoinColumnSQL($tableAlias, $joinColumnName, $quotedColumnName, $isIdentifierColumn, $type)
```

Comment:
This change breaks custom implementations of `AbstractEntityInheritancePersister`. We cannot do this in a bugfix or minor release.

Reply:
I see. It seemed like the obvious change for me. I'm not fully custom with the internal workings of doctrine. Maybe there is a non-breaking solution, which I'm not aware of. As I also stated in the Issue post, that I'm not sure if my change is the correct way of tackling this problem.

Reply:
I've checked back with the core team. The API you're about to change is internal to the ORM, so changing it is fine. Sorry for the confusion.